toolchain/sem_ir/ids.cpp

```diff
@@ -47,6 +47,14 @@ auto ConstantId::Print(llvm::raw_ostream& out, bool disambiguate) const
   }
 }
 
+auto CheckIRId::Print(llvm::raw_ostream& out) const -> void {
+  if (*this == Cpp) {
+    out << Label << "Cpp";
```

Comment:
```suggestion
    out << Label << "(Cpp)";
```

This would be like inst(Singletons) as well

Reply:
Done